sktree/tree/_unsup_criterion.pxd

```diff
@@ -41,8 +39,8 @@ cdef class UnsupervisedCriterion:
     # Unsupervised criterion can be used with splitter and tree methods.
     cdef int init(
         self,
-        const DOUBLE_t[:, ::1] X,
-        const DOUBLE_t[:] sample_weight,
+        const DOUBLE_t[:,:] X,
+        DOUBLE_t* sample_weight,
```

Comment:
Why are we changing it from a Cython memoryview to pointer?

Reply:
Thank you! I'm not sure what happened here, but I changed it back to memoryview